eip-2470.md

```diff
@@ -99,35 +144,35 @@ This contract is going to be deployed using the keyless deployment method---also
+![](data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAIAAAACACAYAAADDPmHLAAAB2UlEQVR4nO3dsU1DARBEQZdCDZRCG0jURg90QEA70ICzC1bLTvBi+9bzsy/58fr583vp7etbwa6/3wOA7gAYD4DxABgPgPEAGA+A8QAYD4DxABgPgPEAGA+A8QAYD4Dx4gA+Xt5PpQdIPwDX/QAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADgGYD2A67fPw0w/QABAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADwPwGkf8Dr57eX3g8AALoPaC+9HwAAdB/QXno/AADoPqC99H4AANB9QHvp/QAAoPuA9tL7AQBA9wHtpfcDAIDuA9pL7wfAOoD0CxHpFzrSpfcDAIDuA9pL7wcAAN0HtJfeDwAAug9oL70fAAB0H9Beej8AAOg+oL30fgAA0H1Ae+n9AACg+4D20vsBAEB+BOUCYDwAxgNgPADGA2A8AMYDYDwAxgNgPADGA2A8AMYDYDwAxgNgvMf1hQR1B8B4AIwHwHgAjAfAeACMB8B4AIwHwHgAjAfAeACMB8B4AIwHwHhnANcXEtJ/uHDt+v2v+wEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADAk/4AAAL56BraQkoAAAAASUVORK5CYII=)
 
-`0xBb6e024b9cFFACB947A71991E386681B1Cd1477D`
+`TBD`
 
 This account is generated by reverse engineering it from its signature for the transaction. 
 This way no one knows the private key, but it is known that it is the valid signer of the deployment transaction.
 
 > To deploy the registry, 0.0247 ether MUST be sent to this account *first*.
 
 ### Factory Contract Address
-![](data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAIAAAACACAYAAADDPmHLAAABn0lEQVR4nO3coRECMRRF0S2GutCUQzd4WqAMLB4qQGWYP+EecXXeZo/OcTrf35Ndbq+l7F/rmB6w+wXuvh+A+H4A4vsBiO8HIL4fgPh+AOL7AYjvByC+H4D4fgDi+wGI7wcgvh+A+H4A4vuXAUxfwPX5GG33+wMAgL0/AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAPgGYHrA9A+cbhoQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA/wlgesD0+bvvXz0fgM33AwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAB8ATB9gZoNgHgAxAMgHgDxAIgHQDwA4gEQD4B4AMQDIB4A8QCIB0A8AOId0w8caK3V/wfA5gEQD4B4AMQDIB4A8QCIB0A8AOIBEA+AeADEAyAeAPEAiAdAPADiARAPgHgAxAMgHgDxAIgHQDwA4gEQD4B4AMQDIB4A8QCItwxg+oECDT8QMT1AAAgAASAABIAAEAACQAAIAAEgAASAANAv+gDxVDRR1CVqRAAAAABJRU5ErkJggg==)
+![](data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAIAAAACACAYAAADDPmHLAAAB1klEQVR4nO3dsY3CABAEQEdUQml0hvSFUAsxISk08B+drNP+TrCx99YT28fr8v5Mcr3/jPJ83EaZPj+9//T9HQBk9wcAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAOAnANqD0bO8PAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADA/wTQPsB2tvcHAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgDNe4PYA29neDwAAsg9Iz/Z+AACQfUB6tvcDAIDsA9KzvR8AAGQfkJ7t/QAAIPuA9GzvBwAA2QekZ3s/AADIPiA92/sB0A5g+sOD7QOmz0/vP31/AIT3BwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD4A0D7gO39ASjvD0B5fwDK+wNQ3h+A8v4AlPcHoLw/AOX9ASjvD0B5fwDK+wNQ3h+A8v4AlPcHoLz/GMD2ANPnp/cHAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgDMATDP9wME2gO0PbAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADAL/kCzPReJEuzwFwAAAAASUVORK5CYII=)
 
-`0xce0042B868300000d44A59004Da54A005ffdcf9f`
+`TBD`
 
 The contract has the address above for every chain on which it is deployed.
-### ABI for SingletonFactory:
+### ABI for ERC2470:
```

Comment:
The fallback method should be in the abi probably (also the method to generate the address)

Reply:
I added the fallback. I didn't included the receive() method, even if compiler exposed it, because its a always fail. I think thats a bug on solidity that is "forcing me" to include the receive().

Reply:
Btw, I removed the receive method, now if someone uses .send or .transfer against this contract, it will fail in the first line of fallback()